doc/CMakeLists.txt

```diff
@@ -1,19 +1,7 @@
-if(HIGH_FIVE_DOCUMENTATION)
-find_package(Pandoc REQUIRED)
-
-configure_file(build.sh.cmake build.sh @ONLY)
-
-add_custom_target(doc ALL DEPENDS
-	COMMAND sh ${CMAKE_CURRENT_BINARY_DIR}/build.sh
-	DEPENDS ${CMAKE_CURRENT_BINARY_DIR}/build.sh
-)
-
-install(
-	FILES ${CMAKE_CURRENT_BINARY_DIR}/index.html
-    DESTINATION ${HTML_INSTALL_DIR}
-)
-install(
-	DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/src
-    DESTINATION ${HTML_INSTALL_DIR}
-)
-endif(HIGH_FIVE_DOCUMENTATION)
\ No newline at end of file
+find_package(Doxygen)
+if(Doxygen_FOUND)
```

Comment:
I would prefer having an option to compile the doxygen docs. If I just want to install the library on several machines I want to have a quick option to do so.